relay-config/src/config.rs

```diff
@@ -589,6 +605,7 @@ impl Default for Http {
             auth_interval: Some(600), // 10 minutes
             outage_grace_period: DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD,
             encoding: HttpEncoding::Gzip,
+            client: HttpClient::Actix,
```

Comment:
Let's make this an internal/undocumented option. We don't want users to use this and just remove it when we're satisfied with the results.

```suggestion
            _client: HttpClient::Actix,
```